### ISSUE TITLE
Splitter does not respect `resolver.extensions`

### ISSUE THREAD

**Issue by buschtoens** [ef4/ember-auto-import#248]
When I configure webpack to support `.ts` files, I still need to append the `.ts` file extension in the `import` statement.

```ts
import bar from 'foo/bar.ts'; // works
import bar from 'foo/bar';    // breaks
```

This is the config. It's not perfect yet and likely misses the `preset-env`, but it's sufficient to load `.ts` files directly from yarn workspaces.

```ts
{
  webpack: {
    resolve: {
      extensions: ['.ts', '.js']
    },
    module: {
      rules: [
        {
          test: /\.ts$/,
          use: {
            loader: 'babel-loader-8',
            options: {
              presets: ['@babel/preset-typescript']
            }
          }
        }
      ]
    }
  }
}
```

This can be fixed by adding `.ts` to the list of extensions in the splitter:

https://github.com/ef4/ember-auto-import/blob/e7d34659666ece3a33c822971e12720680b86c89/packages/ember-auto-import/ts/splitter.ts#L17-L22

The [docs for `enhanced-resolve`](https://github.com/webpack/enhanced-resolve#passing-options-from-webpack) actually suggest that the `webpack.resolve` options should be passed through.

Tangentially related to #74.

I am happy to send in a PR, just let me know, what you think the optimal solution is. I'd suggest passing the `webpack.resolve` through to the splitter and creating individual resolvers for each set of options, in case there are diverging options.

**Comment by ef4** [comment]
We should try to follow how embroider works, see https://github.com/embroider-build/embroider/blob/8bd940d20e83a06733d63bac050c16738723f83c/packages/compat/src/compat-app.ts#L143

I think we can do something similar here. Either always make `ts` resolvable (it doesn't seem harmful) or do it when we see ember-cli-typescript is in the project.

**Comment by buschtoens** [comment]
Thanks for the pointer into the embroider code!

Statically adding `ts` already unblocks me and is minimally invasive, as it does not require to mess around with how and when the resolver is instantiated, which would be necessary, if we'd add it conditionally.

Technically I'd still prefer to be able to explicitly define the extensions, but this sounds like a case of YAGNI.

I'll send a PR right away. Kudos 🙏

**Comment by jamescdavis** [comment]
Resolved in release 1.6.0
